src/ui/mainwindow.cpp

```diff
@@ -495,9 +501,15 @@ MainWindow::MainWindow(Application* app, SystemTrayIcon* tray_icon, OSD* osd,
   connect(this, SIGNAL(NewDebugConsole(Console*)), app_,
           SIGNAL(NewDebugConsole(Console*)));
 
+#ifdef HAVE_BACKGROUND_STREAMS
   background_streams_->AddAction("Rain", ui_->action_rain);
   background_streams_->AddAction("Hypnotoad", ui_->action_hypnotoad);
   background_streams_->AddAction("Make it so!", ui_->action_enterprise);
+#else
+  ui_->action_rain->setVisible(false);
+  ui_->action_hypnotoad->setVisible(false);
+  ui_->action_enterprise->setVisible(false);
```

Comment:
Could these be removed from mainwindow.ui and added to the extras menu dynamically?

Reply:
It would be a cleaner solution indeed, but I'm not sure how to deal with the translations of the menus, given that the translations are managed automatically.

Reply:
I think you can just call tr() like this: https://github.com/clementine-player/Clementine/blob/master/src/ui/mainwindow.cpp#L666

Reply:
Done. I'm still unsure about the translation files because there are remaining `#: ../bin/src/ui_mainwindow.h:748` which don't match (now it's used in `mainwindow.cpp`). I don't know if it could break something. At least it compiles and works fine.